core/src/data/value/json.rs

```diff
@@ -38,6 +38,7 @@ impl TryFrom<Value> for JsonValue {
             Value::Bool(v) => Ok(JsonValue::Bool(v)),
             Value::I8(v) => Ok(v.into()),
             Value::I64(v) => Ok(v.into()),
+            Value::I128(v) => Ok(v.to_string().into()),
```

Comment:
If this happens, why not try to convert the `JsonValue` to a `String`?
Maybe we want a type like `BigInt` or a truncated `Number` type...?
What do @panarch think?

Reply:
You might want to check what I did with this.  Is this what you are after?